src/composite-data/actions/cellar/AAVE_V2_STABLE_CELLAR/fetchCellarShareBalance.ts

```diff
@@ -0,0 +1,31 @@
+import BigNumber from "bignumber.js"
+import { AaveV2CellarV2 } from "src/abi/types"
+
+export const fetchCellarShareBalance = async ({
+  contract,
+  aaveClrBalance,
+  totalBondedAmount,
+}: {
+  contract: AaveV2CellarV2
+  aaveClrBalance?: string
+  totalBondedAmount?: string
+}) => {
+  try {
+    if (!aaveClrBalance) {
+      throw new Error("aaveClrBalance is undefined")
+    }
+    if (!totalBondedAmount) {
+      throw new Error("totalBondedAmount is undefined")
+    }
+
+    const cellarSharebalance = await contract.convertToAssets(
+      new BigNumber(aaveClrBalance).plus(totalBondedAmount).toFixed()
```

Comment:
Only suggestion I'd make here is to define this BigNumber as its own variable above, just helps with readability a little bit :)